src/main.ts

```diff
@@ -62,3 +63,6 @@ useOmniscientLoggerStore()
 
 // Post-pinia connections setup
 setupPostPiniaConnections()
+
+// Setup data lake actions
```

Comment:
Not a huge fan of comments that are just a duplicated of functions calls.

Reply:
Improved the comment to better reflect what happens without being redundant before merging.